tango/integrations/wandb/torch_train_callback.py

```diff
@@ -65,9 +65,9 @@ def __init__(
         super().__init__(*args, **kwargs)
 
         if self.is_local_main_process and "WANDB_API_KEY" not in os.environ:
-            print(
+            warnings.warn(
```

Comment:
Better to use the `warnings` module since these will be redirected to the logging system anyway when logging is configured.